src/FellowOakDicom.Serialization.Forked/GlobalSuppressions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Instead of all of these exemptions, let's just not run our analyzers on this project as it's a fork of something else's code.

As a quality-of-life change, I'd update the existing section related to the analyzers in our `Directory.Build.props` file to be:
```xml
<PropertyGroup>
  <AnalysisLevel>latest-All</AnalysisLevel>
  <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
</PropertyGroup>
```
And for this particular project, override these settings with the following:
```xml
<PropertyGroup>
  <AnalysisLevel></AnalysisLevel>
  <AnalysisMode>None</AnalysisMode>
  <EnforceCodeStyleInBuild>false</EnforceCodeStyleInBuild>
</PropertyGroup>
```